src/main/java/org/apache/maven/plugins/assembly/archive/DefaultAssemblyArchiver.java

```diff
@@ -351,6 +349,23 @@ protected Archiver createArchiver(
         return archiver;
     }
 
+    private void configureJarArchiver(
+            JarArchiver archiver, String mergeManifestMode, MavenArchiveConfiguration configuration) {
+
+        if (mergeManifestMode != null) {
+            archiver.setFilesetmanifest(JarArchiver.FilesetManifestConfig.valueOf(mergeManifestMode));
+        }
+
+        archiver.setMinimalDefaultManifest(true);
```

Comment:
This will set it?

Reply:
yes - exactly

Reply:
So this leaves the user no option to include it?

Reply:
The same is in Maven Archiver.

Reply:
what if mergeManifestMode is set to `skip`?
I find it surprising to have `true` hardcoded

Reply:
> what if mergeManifestMode is set to `skip`? 

I don't test such ...

>I find it surprising to have `true` hardcoded

copied from maven-archived 😄

Reply:
default value for `mergeManifestMode` is `skip`

 `setMinimalDefaultManifest` has only effect on code: https://github.com/codehaus-plexus/plexus-archiver/blob/40b1b12e849ef62021e9d9787add3db7f34267af/src/main/java/org/codehaus/plexus/archiver/jar/Manifest.java#L767

If we omit `addDefaultEntries` or set it with `true` - maven-archiver will add own Manifest 

So IMHO can be hardcoded

Reply:
I definitively don't understand when all this is triggered, but it seems inoffensive, let's go...